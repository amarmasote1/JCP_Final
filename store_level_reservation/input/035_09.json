

{"reservation":[
 {
"storeID" : "0712",
"reservationID" : "20170613221817100252",
"procurementAllowed" : "false",
"shippable" : "true",
"uom" : "EACH",
"itemID" : "12345678051",
"lineID" : "2",
"qty" : "10"
 }

]}
