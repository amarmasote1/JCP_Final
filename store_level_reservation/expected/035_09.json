{"status":200,"message":"Success","body":[{"shortageQty":0,"reservedQty":10,"partiallyReservedQty":0,"problemType":"SUCCESS","orgCode":"JCP","requestLine":{"lineID":"2","itemID":"1234567-8051","uom":"EACH","bopus":false,"shippable":true,"deliveryMethod":null,"increaseCapacity":null,"shutdown":false,"qty":10,"storeID":"0712","channel":null,"reservationID":"20170613221817100252","procurementAllowed":false}}]}